components/runpod/common/utils.mjs

```
@@ -0,0 +1,79 @@
+
+function cleanInput(input) {
+  return Object.fromEntries(
+    Object.entries(input)
+      .filter(([
+        , value,
+      ]) => value !== null && value !== undefined)
+      .map(([
+        key,
+        value,
+      ]) => {
+        if (typeof value === "string" && !isNaN(value)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)